### PR TITLE
Allow rpc method can be overrided

### PR DESCRIPTION
**Pull request by Tommo-L**
Close https://github.com/neo-project/neo-modules/issues/361

### CODE REVIEW

**Review by shargon — COMMENTED** [review]
protected virtual?

**Comment by Tommo-L** [comment]
> protected virtual?

Done

File: src/RpcServer/RpcServer.Wallet.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ namespace Neo.Plugins
 {
     partial class RpcServer
     {
-        private class DummyWallet : Wallet
+        protected class DummyWallet : Wallet
```
Comment (shargon) [inline src/RpcServer/RpcServer.Wallet.cs:26]: I think that we can use private here

Comment (Tommo-L) [reply src/RpcServer/RpcServer.Wallet.cs]: Done

**Review by shargon — APPROVED** [review]

**Comment by superboyiii** [comment]
Tested, works well on mine.

**Review by superboyiii — DISMISSED** [review]

File: src/RpcServer/RpcServer.SmartContract.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ namespace Neo.Plugins
 {
     partial class RpcServer
     {
-        private class Signers : IVerifiable
+        protected class Signers : IVerifiable
```
Comment (shargon) [inline src/RpcServer/RpcServer.SmartContract.cs:22]:
```suggestion
        private class Signers : IVerifiable
```

**Review by shargon — APPROVED** [review]

**Pull request merged**